analyzer/flow_server.go

```diff
@@ -45,39 +45,47 @@ import (
 	"github.com/skydive-project/skydive/topology/graph"
 )
 
+// ErrFlowUDPAcceptNotSupported error the connection can't accept as it's UDP based
 var ErrFlowUDPAcceptNotSupported = errors.New("UDP connection is datagram based (not connected), accept() not supported")
 
+// FlowConnectionType describe an UDP or TLS connection
 type FlowConnectionType int
 
 const (
+	// UDP connection
 	UDP FlowConnectionType = 1 + iota
+	// TLS connection
 	TLS
 )
 
+// FlowServerConn describe a flow server connection
```

Comment:
describes

going to stop here to fix the missing "s" for the third person but please recheck everything